src/v/kafka/security/scram_algorithm.cc

```diff
@@ -0,0 +1,205 @@
+#include "kafka/security/scram_algorithm.h"
+
+#include "utils/base64.h"
+#include "utils/to_string.h"
+#include "utils/utf8.h"
+
+#include <boost/algorithm/string.hpp>
+
+namespace kafka {
+
+// ALPHA / DIGIT / "/" / "+"
+// NOLINTNEXTLINE
+#define BASE64_CHAR "[a-zA-Z0-9/+]"
```

Comment:
use internal ns?

Reply:
but also since macros are global, use REDPANDA_* or RP_*

Reply:
ah good point!

Reply:
exactly. plus, cpp will let us know if we happened to have accidentally redefined